src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/WriteTagCheck.java

```diff
@@ -73,18 +75,40 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
- * To configure the check to print warnings if an "@incomplete" tag is found,
- * and not print anything if it is not found:
+ * Example:
+ * </p>
+ * <pre>
+ * &#47;**
+ * * Valid
```

Comment:
please update to `OK, the check does nothing by default`.
same for `Invalid`.

Reply:
rewrote completely